### ISSUE TITLE
Offload GPU compute!

### ISSUE THREAD

**Issue by gperdrizet** [gperdrizet/llm_detector#38]
Really need to get GPU compute off of local hardware. RunPod seems like  good option, the only rub is we need access to the encoding and the logits from the model. Also, our perplexity ratio KL divergence score is calibrated on output from models loaded in 4-bit via HuggingFace's transformers... Probably want to do the same in the cloud. Might not be a problem, but worth thinking about.

**Comment by gperdrizet** [comment]
Finally some movement on this, wait for an update about external cloud compute resources in a few days. Should be some good news...

**Comment by gperdrizet** [comment]
After receiving Google for Startups Cloud Program grant - will be deploying the whole project there. Closing this issue as won't fix.
